app/src/main/java/com/yoshitoke/weatheringwithyou/utils/extensions.kt

```diff
@@ -0,0 +1,44 @@
+package com.yoshitoke.weatheringwithyou.utils
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+fun Int.unixTimestampToDateTimeString() : String {
+
+    try {
+        val calendar = Calendar.getInstance()
+        calendar.timeInMillis = this*1000.toLong()
+
+        val outputDateFormat = SimpleDateFormat("dd MMM, yyyy - hh:mm a", Locale.ENGLISH)
+        outputDateFormat.timeZone = TimeZone.getDefault() // user's default time zone
+        return outputDateFormat.format(calendar.time)
+
+    } catch (e: Exception) {
+        e.printStackTrace()
+    }
+
+    return this.toString()
+}
+
+fun Int.unixTimestampToTimeString() : String {
+
+    try {
+        val calendar = Calendar.getInstance()
+        calendar.timeInMillis = this*1000.toLong()
+
+        val outputDateFormat = SimpleDateFormat("hh:mm a", Locale.ENGLISH)
+        outputDateFormat.timeZone = TimeZone.getDefault()
+        return outputDateFormat.format(calendar.time)
+
+    } catch (e: Exception) {
+        e.printStackTrace()
+    }
+
+    return this.toString()
+}
```

Comment:
merge to a single function like: Int.unixTimestampToString(format: String)

Reply:
ok